src/components/editors/fontEditor.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import { Grid, Typography } from '@material-ui/core'
+
+import { OPTIONS } from '../../lib/options'
+import { Slider, ColorPicker, PopoverIcon } from '../SettingComponent'
+
+const FONT_EDITOR_OPTIONS = [
+  {
+    name: 'Primary font',
+    color: OPTIONS.primaryFontColor,
+    size: OPTIONS.primaryFontSize,
+    larivaar: OPTIONS.primaryLarivaarAssistColor,
+    drop: OPTIONS.primaryDropColor,
+  },
+  {
+    name: 'Secondary font',
+    color: OPTIONS.secondaryFontColor,
+    size: OPTIONS.secondaryFontSize,
+    larivaar: OPTIONS.secondaryLarivaarAssistColor,
+    drop: OPTIONS.secondaryDropColor,
+  },
+]
+
+const FontEditor = () => (
+  <Grid
+    container
+    direction="column"
+    justify="flex-start"
+    alignItems="flex-start"
+    spacing={5}
+  >
+    {FONT_EDITOR_OPTIONS.map( opt => (
```

Comment:
same comments as background editor.

I need to look more closely, but is there a point of refactoring between the two? are they similar?